docs/examples/State Estimation Example.ipynb

```diff
@@ -36,25 +36,10 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "id": "ae11dc9a",
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "C:\\Users\\AL29374\\AppData\\Local\\Temp\\ipykernel_16184\\2477656666.py:3: DeprecationWarning: \n",
-      "Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),\n",
```

Comment:
something like this should do the trick

```py
import warnings

with warnings.catch_warnings(action="ignore", category=DeprecationWarning):
    # suppress warning about pyarrow as future required dependency
    import pandas
```
or otherwise
```py
import warnings

with warnings.catch_warnings():
    # suppress warning about pyarrow as future required dependency
    warnings.filterwarnings("ignore", category=DeprecationWarning)
    import pandas
```